src/SpectralDataManager.cpp

```diff
@@ -0,0 +1,146 @@
+#include "FFT.h"
```

Comment:
Add authorship comment at the head of this and the corresponding .h file, as you have elsewhere with the other new files.